src/app/profile/profile/profile-species.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { MatSelectChange } from '@angular/material/select';
+import { ActivatedRoute, Router } from '@angular/router';
+import { BehaviorSubject, combineLatest, Observable, ReplaySubject } from 'rxjs';
+import { catchError, first, map, switchMap } from 'rxjs/operators';
+import { IndividualPhenophase } from 'src/app/individual/individual-phenophase';
+import { IndividualService } from 'src/app/individual/individual.service';
+import { MasterdataService } from 'src/app/masterdata/masterdata.service';
+import { AuthService } from '../../auth/auth.service';
+import { BaseDetailComponent } from '../../core/base-detail.component';
+import { NavService } from '../../core/nav/nav.service';
+import { PublicUser } from '../../open/public-user';
+import { PublicUserService } from '../../open/public-user.service';
+
+@Component({
+  templateUrl: './profile-species.component.html',
+  styleUrls: ['./profile-species.component.scss']
+})
+export class ProfileSpeciesComponent extends BaseDetailComponent<PublicUser> implements OnInit {
+  latestIndividualObservations$: Observable<IndividualPhenophase[]>;
+  selectedYear = new ReplaySubject<number>(1);
+
+  species = new BehaviorSubject('');
+
+  constructor(
+    private navService: NavService,
+    protected route: ActivatedRoute,
+    private publicUserService: PublicUserService,
+    public dialog: MatDialog,
+    private authService: AuthService,
+    private individualService: IndividualService,
+    private masterdataService: MasterdataService,
+    protected router: Router
+  ) {
+    super(publicUserService, route, router);
+  }
+
+  ngOnInit(): void {
+    super.ngOnInit();
+    this.navService.setLocation('Species - ');
+    this.masterdataService.phenoYear$.pipe(first()).subscribe(a => this.selectedYear.next(a));
+    this.latestIndividualObservations$ = combineLatest([
+      this.selectedYear,
+      this.getDetailId(),
+      this.getSpeciesRouteParameter()
+    ]).pipe(
+      switchMap(([phenoYear, userId, species]) => {
+        console.log('switchMap', { phenoYear, userId, species });
+        return this.individualService
+          .getIndividualPhenohases(
+            this.individualService.listByUserAndSpecies(userId, phenoYear, species.toUpperCase())
+          )
+          .pipe(map(individuals => individuals.sort((l, r) => l.individual.name.localeCompare(r.individual.name))));
+      })
+    );
+    this.latestIndividualObservations$.subscribe(l => console.log({ latestIndividualObservations: l }));
+    this.latestIndividualObservations$
```

Comment:
Let's try not to subscribe if possible as it is error prone and we forget to unsubscribe. Can we assign the observable to this.species$ and use replay() if necessary.